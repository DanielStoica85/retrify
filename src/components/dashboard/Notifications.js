import React from 'react';

const Notifications = () => {
    return (
        <div>
            <p>Notifications</p>
            <p>
                You are running this application in{' '}
                <b>{process.env.NODE_ENV}</b>.
            </p>
        </div>
    );
};

export default Notifications;
