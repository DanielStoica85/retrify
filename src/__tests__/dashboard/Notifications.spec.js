import React from 'react';
import renderer from 'react-test-renderer';
import Notifications from '../../components/dashboard/Notifications';

describe('Notifications component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<Notifications />).toJSON();
        expect(component).toMatchSnapshot();
    });
});
