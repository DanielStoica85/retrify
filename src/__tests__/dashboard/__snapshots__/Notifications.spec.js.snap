// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications component renders correctly 1`] = `
<div>
  <p>
    Notifications
  </p>
</div>
`;
